decimal64const.go

```diff
@@ -40,6 +40,7 @@ const decimal64Base = 1000 * 1000 * 1000 * 1000 * 1000
 const maxSig = 10*decimal64Base - 1
 
 const expOffset = 398
+const minExp = -398
```

Comment:
Let's clean this up.
1. Use consistent prefixes.
2. Suffix everything with `64`.
3. Show the math behind the values.

Thus:
```go
const (
	decDigits64  = 16
	expBits64    = 10
	expRange64   = 3 << (expBits - 2)
	symmMaxExp64 = expRange64 / 2
	symmMinExp64 = 1 - symmMaxExp64
	maxExp64     = symmMaxExp64 - (decDigits64 - 1)
	minExp64     = symmMinExp64 - (decDigits64 - 1)
)
```
We end up with `symm(Max|Min)Exp64` = 384|-383 and `(max|min)Exp64` = 369|-398.

The decimal128 group can then be written in exactly the same form, with the only changes being to the first two constants:
```go
	decDigits128  = 34
	expBits128    = 14
```
It could even be extended to decimal32, in the event that we ever wanted to implement it:
```go
	decDigits32  = 7
	expBits32    = 8
```

----

Technically, we could even derive `decDigits(32|64|128)`:
```go
	decDigits64  = (64 - expBits64) * 3 / 10
```
`64 - expBits64` is the number of bits of precision available for the significand and 3/10 is close enough to log(2)/log(10) to convert bits to digits for the given values of `expBits(32|64|128)`. But I think this a bit too abstruse for the reader.

Reply:
https://play.golang.org/p/dhb9HB92rcK

Reply:
Moved this into an issue https://github.com/anz-bank/decimal/issues/51